### ISSUE TITLE
Indentation problem in online document

### ISSUE THREAD

**Issue by oeway** [fchollet/keras#1682]
The indentation is not displaying correctly in the online document.

![screen shot 2016-02-10 at 10 44 14](https://cloud.githubusercontent.com/assets/478667/12943701/45b06eec-cfe3-11e5-9f68-f3e10fabf7aa.png)
http://keras.io/models/

And the corresponding docstring in the source code is:
```
        # Examples
        ```python
            def generate_arrays_from_file(path):
                while 1:
                    f = open(path)
                    for line in f:
                        # create numpy arrays of input data
                        # and labels, from each line in the file
                        x, y = process_line(line)
                        yield x, y
                    f.close()
            model.fit_generator(generate_arrays_from_file('/my_file.txt'),
                                samples_per_epoch=10000, nb_epoch=10)
        ```
```

**Comment by parag2489** [comment]
That is more like a psuedo-code. Look at #1638 for a code snippet on how to make a proper data generator.

**Comment by oeway** [comment]
My point is not how to make a generator, it's the document is not generated properly. The document is generated from the docstring in the code.

For example, you can see the following code in the file: https://github.com/fchollet/keras/blob/657b9fb48e93b59083d2e0b8a5e4daf237179dbc/keras/models.py

```
        # Examples
        ```python
            def generate_arrays_from_file(path):
                while 1:
                    f = open(path)
                    for line in f:
                        # create numpy arrays of input data
                        # and labels, from each line in the file
                        x, y = process_line(line)
                        yield x, y
                    f.close()
            model.fit_generator(generate_arrays_from_file('/my_file.txt'),
                                samples_per_epoch=10000, nb_epoch=10)
        ```
```

**Comment by parag2489** [comment]
Oh okay, when you said, _The indentation is not working correctly in the online document._. I thought when you convert that psuedo-code into an actual one, it was not working for you.

**Comment by oeway** [comment]
Thanks all the same and sorry for the misleading.

**Comment by fchollet** [comment]
Any other places in the documentation where you're seeing this?

**Comment by fchollet** [comment]
This is fixed.
